data/frm/Crecy_1997_yPonthus/stemma.gv

```diff
@@ -0,0 +1,19 @@
+digraph {
+
+y [color="grey"];
+y -> y1;
+y -> y2; 
+
+y1 [color="grey"];
+y1 -> B;
+y1 -> C2; 
+y1 -> L1; 
+y1 -> P9;
+
+y2 [color="grey"];
+y2 -> P6; 
```

Comment:
Idem ?

Reply:
The common ancestor of all these mss. is y2, and they are on the same level.